app/src/main/java/com/texthip/thip/ui/group/note/viewmodel/GroupNoteAiViewModel.kt

```
@@ -0,0 +1,76 @@
+                        )
+                    }
+                }
+                .onFailure { throwable ->
+                    val errorMessage = when (throwable) {
+                        is HttpException -> {
+                            val errorBody = throwable.response()?.errorBody()?.string()
+                            if (errorBody != null) {
+                                try {
+                                    val errorResponse = json.decodeFromString<BaseResponse<Unit>>(errorBody)
+                                    errorResponse.message
+                                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)